alembic.ini

```diff
@@ -71,7 +71,7 @@ hooks=ruff_check, ruff_format
 
 ruff_check.type = exec
 ruff_check.executable = ruff
-ruff_check.options = check --fix REVISION_SCRIPT_FILENAME
+ruff_check.options = check --ignore F401 --fix REVISION_SCRIPT_FILENAME
```

Comment:
This addition is really useful!
Could you maybe add a comment explaining why we don't want missing imports to be removed during the migration generation?